addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1543,22 +1543,26 @@ const Wysiwyg = Widget.extend({
                 return;
             }
             const $image = $(this.lastMediaClicked);
+            const imgTransformBtn = this.toolbar.el.querySelector('#image-transform');
             if ($image.data('transfo-destroy')) {
                 $image.removeData('transfo-destroy');
                 return;
             }
             $image.transfo({document: this.odooEditor.document});
+            imgTransformBtn.setAttribute('title', _t('Click again to reset transformation'));
```

Comment:
@lebl-odoo Since it's a new string, it won't have a translation until someone translates it and the user updates their database. For that reason, I would advise to keep `Transform the picture (click twice to reset transformation)` in the two cases in stable. If you'd rather pay the price of not having it translated for a while (it will be translated eventually), let me know and I'll merge the PR as is.

Reply:
@dmo-odoo Is it possible to keep the old string for stable versions and use the new one for master version? If not, I'll live with it and eventually add this change in another task (I won't create a new one for a copy issue)

Reply:
@lebl-odoo Yes, it's possible. @adsa-odoo Could you remove that new string from stable version please? The button will keep the same text regardless of active state. You'll put the new string in the master forward-port.

Reply:
@dmo-odoo Sure, I've updated the code, I'll add a new commit in master fw-port.